src/components/Navbar.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const logoAnimation = keyframes`
+  from {
+    width: 0px;    
+  }
+  to{
+    width: 68px;
+    background-color: #dedede;
+  }
+`;
+
+const showContainerAnimation = keyframes`
+  from {
+    opacity: 0
+  }
+  to{
+    opacity: 1
+  }
+`;
+
+export default function Navbar() {
+  return (
+    <Container>
+      <Logo>&gt; we-record</Logo>
+      <div>
+        <GoToMyPageBtn>마이 페이지</GoToMyPageBtn>
+        <GoToBatchBtn>기수 페이지</GoToBatchBtn>
+        <LogoutBtn>로그 아웃</LogoutBtn>
+      </div>
+    </Container>
+  );
+}
+
+const Container = styled.nav`
+  ${({ theme }) => theme.flexbox('row', 'space-between')}
+  padding: 12px 24px;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.white};
+  animation-name: ${showContainerAnimation};
+  animation-duration: 1s;
```

Comment:
animation에 관련된 값들을 하나로 합치는 것이 효율적일까요? 아니면 용민님이 하신 것처럼 세분화해서 선언하는게 좋을까요???

Reply:
음.. 저도 처음에 사용했을때 합쳐서 사용했는데 가독성이 좀 떨어지더라구요!! 
```suggestion
  animation-name: ${showContainerAnimation} 1s
```
이런식으로 사용하게 되면 한 줄에 표현은 가능하지만, 컨벤션 순서를 잘 모르면 어떤 값이 어떤 속성을 가리키는지 구별하기가 어려워서, 일부러 나누어 표기했습니다!!